deployment/crontab

```diff
@@ -20,7 +20,8 @@
 
 * * * * * echo "Started RG" > /proc/1/fd/1 2>/proc/1/fd/2
 
-* * * * * /usr/bin/flock -n /app/result-gen.lock /usr/local/bin/python3 /app/src/result_gen_with_api.py --scan_parent_dir /app/data/scans/ --blur_workflow_path /app/src/workflows/blur-workflow.json --depthmap_img_workflow_path /app/src/workflows/depthmap-img-workflow.json --standing_laying_workflow_path /app/src/workflows/standing_laying-workflow.json --height_workflow_artifact_path /app/src/workflows/height-workflow-artifact.json --height_workflow_scan_path /app/src/workflows/height-workflow-scan.json --height_MCNN_workflow_path /app/src/workflows/height-MCNN-workflow.json --weight_workflow_artifact_path /app/src/workflows/weight-workflow-artifact.json --weight_workflow_scan_path /app/src/workflows/weight-workflow-scan.json > /proc/1/fd/1 2>/proc/1/fd/2
+* * * * * /usr/bin/flock -n /app/result-gen.lock /usr/local/bin/python3 /app/src/result_gen_with_api.py --scan_parent_dir /app/data/scans/ --blur_workflow_path /app/src/workflows/blur-workflow.json --depthmap_img_workflow_path /app/src/workflows/depthmap-img-workflow.json --standing_laying_workflow_path /app/src/workflows/standing_laying-workflow.json --height_workflow_artifact_path /app/src/workflows/height-plaincnn-workflow-artifact.json --height_workflow_scan_path /app/src/workflows/height-plaincnn-workflow-scan.json --height_depthmapmultiartifactlatefusion_workflow_path /app/src/workflows/height-depthmapmultiartifactlatefusion-workflow.json --weight_workflow_artifact_path /app/src/workflows/weight-workflow-artifact.json --weight_workflow_scan_path /app/src/workflows/weight-workflow-scan.json > /proc/1/fd/1 2>/proc/1/fd/2
```

Comment:
I would like to make this easier to read somehow. Can you try if this suggestion works?

```suggestion
* * * * * /usr/bin/flock -n /app/result-gen.lock /usr/local/bin/python3 /app/src/result_gen_with_api.py \
    --scan_parent_dir /app/data/scans/ \
    --blur_workflow_path /app/src/workflows/blur-workflow.json \
    --depthmap_img_workflow_path /app/src/workflows/depthmap-img-workflow.json \
    --standing_laying_workflow_path /app/src/workflows/standing_laying-workflow.json \
    --height_workflow_artifact_path /app/src/workflows/height-plaincnn-workflow-artifact.json \
    --height_workflow_scan_path /app/src/workflows/height-plaincnn-workflow-scan.json \
    --height_depthmapmultiartifactlatefusion_workflow_path /app/src/workflows/height-depthmapmultiartifactlatefusion-workflow.json \
    --weight_workflow_artifact_path /app/src/workflows/weight-workflow-artifact.json \
    --weight_workflow_scan_path /app/src/workflows/weight-workflow-scan.json \
    > /proc/1/fd/1 2>/proc/1/fd/2
```